src/lib.rs

```diff
@@ -232,13 +242,15 @@ impl Config {
                                          .debug(false)
                                          .target(&target)
                                          .host(&host)
+                                         .static_crt(self.static_crt.unwrap_or(false))
```

Comment:
For maximal safety to trigger the `None` fallback logic in gcc-rs, could this only be configured if `static_crt` is explicitly set in cmake?